cli-typescript/src/utils/auth/okta.ts

```diff
@@ -0,0 +1,192 @@
+/**
+ * Authenticate using okta
+ *
+ * TODO: we forked this file from cli/ to unblock eth tooling. Refactor this into a shared lib.
+ */
+
+import axios from 'axios';
+import { createHash, randomBytes } from 'crypto';
+import express from 'express';
+import open from 'open';
+import { stringify } from 'querystring';
+
+export const OKTA = {
+  oktaOrgUrl: 'https://magiceden.okta.com',
+  clientId: '0oaczwrqrdhJTKiuC696',
+  scopes: 'openid email profile offline_access',
+};
+
+const redirectUri = 'http://localhost:30033/login/callback';
```

Comment:
Can we be sure this port selection will be consistent?

Reply:
Yes. the redirect server is served [here](https://github.com/rovilay/magicdrop/blob/d7548bf1f75c711ab17a5c226a42a96750477cfd/cli-typescript/src/utils/auth/okta.ts#L106-L107)